backend/api/models.py

```diff
@@ -0,0 +1,48 @@
+import uuid
+from django.db import models
+from django.utils import timezone
+
+
+# Create your models here
+class Place(models.Model):
+    address = models.CharField(max_length=55, primary_key=True)
+    name = models.CharField(max_length=55)
+
+
+class Category(models.Model):
+    category_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    name = models.CharField(max_length=55)
+
+
+class Cashflow(models.Model):
+    cashflow_id = models.UUIDField(primary_key=True, default=uuid.uuid4, editable=False)
+    is_income = models.BooleanField()
+    overall_value = models.FloatField()
+    timestamp = models.DateTimeField(default=timezone.now)
+    categories = models.ManyToManyField(Category)
```

Comment:
I hope I have resolved your question, here?

Reply:
Im not sure if this is what we want, becaus in our ER-Diagramm the Relation between Cashflow and Category is one to many and not many to many. The question was related to the "models.DO_NOTHING" 😅